presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/TransformCorrelatedJoinToJoin.java

```diff
@@ -15,21 +15,34 @@
 
 import com.google.common.collect.ImmutableList;
```

Comment:
could you add more description to commit message? what does it fix specifically?